### ISSUE TITLE
Problems encountered during training

### ISSUE THREAD

**Issue by 49xxy** [genforce/eqgan#2]
Thank you very much for your excellent work！
I encountered the following problems during training.Hope to get your help.
![cd491a6fe034740a33ae6262eb4e2ac](https://user-images.githubusercontent.com/102204219/179433503-1a3586d9-b604-4741-9938-aeae5be9ce75.png)

**Comment by 49xxy** [comment]
How can I get figure 2 in the article? What about some of the training options?
![image](https://user-images.githubusercontent.com/102204219/179434388-4ed0a72f-6258-43a0-90fc-dad7d02437fd.png)

**Comment by jytime** [comment]
Hi,

The error was raised since the module torchvision was not installed in your environment. It should be installed together with Pytorch.

Figure 2 just needs a pretrained StyleGAN2 model and an implementation of GradCAM. You can implement GradCAM like the function [_compute_GradCam_multi( )_](https://github.com/genforce/eqgan-sa/blob/92bf87ba2b906f8c84be721036c5d68febcb0afa/training/loss.py#L149) in our loss.py, or some other public implementations. The results would be the same.

Best,
Jianyuan

**Comment by jytime** [comment]
I close this due to no activity. Please feel free to raise a new issue if you have any other questions.
